_hackbars/pratham.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: hackbar
+title: DyingBeauty
+author: Pratham
+---
+
+Open source is saviour for the lazy and talented. People love to work to make your project better. :)
+
+---
+
+Just imagine the world without those red roses and without those alphonso mangoes. Tough to imagine ain't it? Let's then do something so that world never runs out of mangoes.
```

Comment:
hey @praX03, need to change some styles and typos -

```suggestion
## Just imagine the world without those red roses and without those Alphonso mangoes. Tough to imagine ain't it? Let's then do something so that the world never runs out of mangoes.
```